src/pages/index.tsx

```diff
@@ -0,0 +1,152 @@
+        <Grid container className="grid-container" justify="center" alignItems="flex-start" spacing={4}>
+        
+          <Grid item className="grid-box"><VerticalSpacer size={84} />
+            <Typography color="primary" variant="subtitle1">
+              DESIGNED BY DEVELOPERS
+            </Typography>
+            <VerticalSpacer size={12} />
+            <Typography color="textPrimary" variant="h1">
+              Powerful developer tools with a simple user-experience
+            </Typography>
+            <VerticalSpacer size={30} />
+            <Grid
+              container
+              direction="row"
+              justify="flex-start"
+              alignItems="flex-start"
+              spacing={2}
+            >
+              <Grid item xs>
+                <Typography color="textPrimary" variant="subtitle1">
+                  Easy Build & Deployment
+                </Typography>
+                <Typography color="textSecondary" variant="subtitle2">
+                  Simply connect your repo and you're off! KintoHub can build
+                  and deploy your code as-is.
+                </Typography>
+                <VerticalSpacer size={10} />
+                <Typography color="textPrimary" variant="subtitle1">
+                  Logs
+                </Typography>
+                <Typography color="textSecondary" variant="subtitle2">
+                  Drill down and debug your issues with real time logs.
+                </Typography>
+              </Grid>
+
+              <Grid item xs>
+                <Typography color="textPrimary" variant="subtitle1">
+                  Advanced Configurations
+                </Typography>
+                <Typography color="textSecondary" variant="subtitle2">
+                  Click your way to cost effective development or scalable production
+                  environments. 
+                </Typography>
+                <VerticalSpacer size={10} />
+                <Typography color="textPrimary" variant="subtitle1">
+                  Command Line Interface (CLI)
+                </Typography>
+                <Typography color="textSecondary" variant="subtitle2">
+                  Use our CLI tools to connect tunnel to your secure databases.
+                </Typography>
+              </Grid>
+            </Grid>
+          </Grid>
+
+          <Grid item>
+            <VerticalSpacer size={84} />
+            <img src={KintoDeveloperTools}></img>
+          </Grid>
+        </Grid>
+      </MuiThemeProvider>
+    </StyledFeatureDescriptionContainer>
+  )
+}
+
+class HomePage extends React.Component {
+  render() {
+    return (
+      <Layout>
+        <SEO
+          title="KintoHub - The best way to deploy fullstack apps"
+          keywords={[
+            `kintohub`,
+            `micro-services`,
+            `backend`,
+            `api`,
+            `deployment`,
+            `vercel`,
+            `render`,
+            `heroku`,
```

Comment:
action: no need to specify here. we can let the SEO to load the static query and the data should be set via `gatsby-config`